frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -126,7 +193,7 @@ export const initSegment = async (props: {
     );
     const anonymousIDArray = Array.from(new Uint8Array(anonymousIDBuffer));
     const anonymousID = anonymousIDArray.map((b) => b.toString(16).padStart(2, '0')).join('');
-    fireTrackingEvent('identify', { anonymousID });
+    fireIdentifyEvent({ anonymousID, canCreateProjects: canCreate, canDelete, canUpdate, isAdmin });
```

Comment:
Why did you rename "canCreate" and not the other props?

Reply:
Also a leftover from when this started. I've changed this now.